sonar-plugin-api/src/main/java/org/sonar/api/server/rule/RulesDefinitionAnnotationLoader.java

```diff
@@ -40,7 +40,9 @@
  *
  * @see org.sonar.api.server.rule.RulesDefinition
  * @since 4.3
+ * @deprecated since 6.0 sonar-check-api is deprecated
  */
+@Deprecated
 public class RulesDefinitionAnnotationLoader {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)